en/frames_rover/ackermann_rover.md

```diff
@@ -112,46 +86,25 @@ Therefore these two parameters have to be set for the slew rates to work!
 
```

Comment:
Above it says "two parameters have to be set for the slew rates to work!". I assume you mean slew rate limiting to work. Again, worth saying here what happens if you get it wrong. I.e. "Too high slew rate settings mean that the vehicle will skid at higher speeds and faster turns", or to put it another way, assume I'm an idiot "why do I care about slew rates", "why am I setting these"